massa-api/src/public.rs

```diff
@@ -33,7 +33,7 @@ use massa_models::{
     block_id::BlockId,
     clique::Clique,
     composite::PubkeySig,
-    config::CompactConfig,
+    config::{CompactConfig, CHAINID},
```

Comment:
let's not import constants directly but pass them through `APIConfig` for example (see how we do for other constants like THREAD_COUNT or VERSION)

Reply:
Fixed